prov/verbs/src/ep_rdm/verbs_rdm_cm.c

```diff
@@ -615,21 +617,76 @@ ssize_t fi_ibv_rdm_conn_cleanup(struct fi_ibv_rdm_conn *conn)
 	return ret;
 }
 
+static int fi_ibv_rdm_poll_cq(struct fi_ibv_rdm_ep *ep)
```

Comment:
Thank you! Avoidance of duplication sounds reasonable, but we don't need to process postponed send queue since it's impossible to send smth in case of peer issued disconnection. I guess, more suitable to wrap part of code from `wc_error` to separate function and also replace duplicated code by `fi_ibv_rdm_process_send_wc` function and newly added one. This is just to don't take away `static inline` definition of `fi_ibv_rdm_tagged_poll_send `and `fi_ibv_rdm_process_send_wc`.